boards/arm/actinius_icarus/actinius_icarus_common_2_0_0.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 Actinius
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default_v2: uart0_default_v2 {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 9)>,
+				<NRF_PSEL(UART_RX, 0, 6)>;
+		};
+	};
+
+	uart0_sleep_v2: uart0_sleep_v2 {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 9)>,
+				<NRF_PSEL(UART_RX, 0, 6)>;
+			low-power-enable;
+		};
+	};
+};
+
+&uart0 {
+	status = "okay";
+
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default_v2>;
+	pinctrl-1 = <&uart0_sleep_v2>;
+	pinctrl-names = "default", "sleep";
+};
+
+&spi3 {
+	status = "okay";
+
+	cs-gpios = <&gpio0 25 GPIO_ACTIVE_LOW>;
+
+	w25q64: w25q64jv@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
+		label = "W25Q64";
+		jedec-id = [ef 40 17];
+		size = <0x4000000>;
+		has-dpd;
+		t-enter-dpd = <3500>;
+		t-exit-dpd = <3500>;
+	};
+};
+
+/ {
+	charger_enable: charger-enable {
+		compatible = "actinius,charger-enable";
+		gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		charger = "auto";
+	};
+};
```

Comment:
can't regulator be used? grep for `regulator-fixed` in some boards.

Reply:
I guess it technically could, but I don't think it's fitting, because this is not a device that is on or off; the charger can be disabled, otherwise it has it's own logic on when it is on (i.e. why the `auto` enum value).